autosubmit/monitor/monitor.py

```diff
@@ -436,38 +511,25 @@ def generate_output(self, expid, joblist, path, output_format="pdf", packages=No
 
             # If the txt has been generated, don't make it again.
             if output_format != "txt":
-                self.generate_output_txt(expid, joblist, path, "default")
+                self.generate_output_txt(expid, joblist, path, True)
```

Comment:
Wrong parameter type (was evaluating to `True`, but because it's a non-empty string)